README_dependencies.md

```diff
@@ -29,18 +29,23 @@ RefreshEnv.cmd # reload the environment
 ```
 
 ### Necessary Dependencies
-1. A C++ compiler that supports C++17.
+1. A C++ compiler that supports C++20.
 See [cppreference.com](https://en.cppreference.com/w/cpp/compiler_support)
 to see which features are supported by each compiler.
 The following compilers should work:
 
-  * [gcc 7+](https://gcc.gnu.org/)
+  * [gcc 11+](https://gcc.gnu.org/)
 	<details>
 	<summary>Install command</summary>
 
-	- Debian/Ubuntu:
+	- Ubuntu 21.04+:
+
+			sudo apt install gcc-11
 
-			sudo apt install build-essential
+	- Ubuntu 18.04-20.04:
+
+			sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test && sudo apt-get update -qq
+			sudo apt install gcc-11
 
 	- Windows:
 
```

Comment:
The next line here tells you how to install mingw on Windows. Since we are deliberately discontinuing support for mingw, maybe we should remove that section, and note the reasons why mingw is unsupported.

Reply:
```suggestion

		The most popular Windows port of `gcc` is `mingw`, which you can install with `choco`.
		Please be aware that `mingw` is not officially supported by this project, owing to numerous 
		longstanding bugs. Use it at your own risk.

```